api/matlab/arrus/DuplexDisplay.m

```diff
@@ -10,8 +10,16 @@
     %   limits to apply (optional name-value argument)
     % :param powerThreshold: power limit [dB] below which the color data \
     %   is not displayed (scalar) (optional name-value argument)
+    % :param turbuThreshold: turbulence limit over which the color data \
+    %   is not displayed (scalar) (optional name-value argument)
+    % :param stdevThreshold: limit of standard deviation of color data \
+    %   over which the color data is not displayed \
+    %   (scalar) (optional name-value argument)
+    % :param thresholdSmoothe: size [pix] of the smoothing kernel applied \
```

Comment:
What type of kernel is used here (simple L2 norm)? (update docs)

Reply:
There was a bug in the kernel definition. It was not meant to be L2 norm, just a circle of ones (1 if euclidean distance is <= some value, 0 otherwise). It's already fixed.

Reply:
Docs also updated, hope the "circular kernel" sounds clear.